vmware_dvs/agent/dvs_neutron_agent.py

```diff
@@ -115,21 +113,18 @@ def delete_network_postcommit(self, current, segment):
             LOG.info(_LI('Network %(id)s not deleted. Reason: %(reason)s') % {
                 'id': current['id'],
                 'reason': e.message})
-        except exceptions.InvalidNetwork:
-            pass
         else:
             dvs.delete_network(current)
+            self.sg_agent.firewall.remove_network(current['id'])
```

Comment:
Bug: do remove_network call for vcenter_firewall engine only. Noop will take exception for remove_network method not found.